apollo-router/src/configuration/mod.rs

```diff
@@ -191,6 +191,11 @@ pub struct Configuration {
     /// Batching configuration.
     #[serde(default)]
     pub(crate) batching: Batching,
+
+    /// Type conditioned fetching configuration.
+    /// If you don't know what this is about, you probably don't need it.
```

Comment:
I feel like there could be a better comment for this. :)

Reply:
XD fair, removing the second line

Reply:
fixed in d14d077d0